algorithms/appo/tests/test_model.py

```diff
@@ -11,8 +12,10 @@
 
 
 class TestModel(TestCase):
-    def test_forward_pass(self):
-        env_name = 'atari_breakout'
+
+    @staticmethod
+    def forward_pass(device_type):
+        env_name = 'doom_benchmark'
```

Comment:
I deliberatly edited this test to not require doom, so it can be run without installing any additional envs. This merge will fail because we both edited the same function. Please merge master into your branch (using my changes, e.g. atari and different obs resolution) and re-submit the pull request.